src/test/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannelIT.java

```diff
@@ -535,10 +554,19 @@ public void testPartialBatchChannelInvalidationIngestion_schematization() throws
     service.closeAll();
   }
 
+  private Map<String, String> getConfForStreaming() {
+    Map<String, String> config = TestUtils.getConfForStreaming();
+    String enableSingleBufferValue = String.valueOf(!useDoubleBuffer);
```

Comment:
It would be easier to use a `useSingleBuffer` for the name and not negate the value. There is an unneeded translation needed.

Reply:
I preferred to have a _turning on_ parameter instead of _turning off_, as our desired behavior is to have a single buffer, but I've changed it as requested for simplicity.